### PR TITLE
LPD-34378 - Email Address api (part 2)

### PR DESCRIPTION
**Pull request by matyaswollner**
https://liferay.atlassian.net/browse/LPD-34378

cc @smottal

### CODE REVIEW

**Comment by matyaswollner** [comment]
cc @smottal

**Comment by matyaswollner** [comment]
ci:test:relevant

File: modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/EmailAddressResourceImpl.java [hunk 1]
```diff
@@ -120,6 +162,68 @@ public Page<EmailAddress> getUserAccountEmailAddressesPage(
...
+
+		if (serviceEmailAddress == null) {
+			throw new NoSuchEmailAddressException();
+		}
+
+		return EmailAddressUtil.toEmailAddress(
+			_updateEmailAddress(serviceEmailAddress, emailAddress));
+	}
+
+	private Long _toListTypeId(String className, String emailAddressType) {
```
Comment (smottal) [inline modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/EmailAddressResourceImpl.java:194]: _getListTypeId

Comment (smottal) [reply modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/resource/v1_0/EmailAddressResourceImpl.java:194]: and emailAddressType -> name

**Review by smottal — CHANGES_REQUESTED** [review]

**Comment by matyaswollner** [comment]
ci:stop

**Comment by matyaswollner** [comment]
ci:test:sf

**Comment by matyaswollner** [comment]
ci:test:relevant

**Review by smottal — APPROVED** [review]

**Comment by smottal** [comment]
ci:report:21679585

**Comment by smottal** [comment]
ci:forward:force

**Pull request closed without merging**